public/nginx.conf

```diff
@@ -128,6 +139,8 @@ http {
             alias   /usr/share/nginx/html/static;
             expires 1y;
             add_header Cache-Control "public";
+            add_header X-Frame-Options SAMEORIGIN;
+            add_header X-Content-Type-Options nosniff;
```

Comment:
A config validator called out that these might be missing when serving static content